src/ct/ct_actions_file.cc

```diff
@@ -387,6 +387,14 @@ void CtActions::preferences_export()
     _pCtConfig->write_to_file(filepath);
 }
 
+void CtActions::command_selnode()
+{
+    auto id = CtDialogs::dialog_selnode(_pCtMainWin);
+    auto node_iter = _pCtMainWin->get_tree_store().get_node_from_node_id(id);
```

Comment:
as you can return an invalid id (-1) for example if the user presses cancel on the dialog, you have to check that the returned node_iter is valid `if (node_iter )` or set_cursor_safe() with a bad input will crash

Reply:
D'oh... Completely slipped my mind on that one. Even added the -1 *for this very purpose*